identity-service/src/models/usertracklisten.js

```diff
@@ -1,6 +1,6 @@
 'use strict'
 module.exports = (sequelize, DataTypes) => {
-  const UserTrackListen = sequelize.define('UserTrackListen', {
+  const UserTrackListens = sequelize.define('UserTrackListens', {
```

Comment:
i believe the singular is actually best practice. sequelize automatically pluralizes these model names

Reply:
undid